Project 3 Lol

```diff
@@ -0,0 +1,350 @@
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  - Sally_arm instance_of fiat object part
+  - Sally_body instance_of object
+  - Sally_arm part_of Sally_body
+  - Sally_arm disjoint_with Limbless_Sally
+  ...
+  - Sally_limbloss instance_of process 
+  - Tuesday instance_of one-dimensional temporal region
+  - Wednesday instance_of one_dimensional temporal region
+  ...
+ - Sally_body involved_in Sally_limbloss
+ - Sally_limbloss has_output Limbless_Sally 
+ - Sally_limbloss occurs_in Wednesday
+ - Tuesday precedes Wednesday
+  
+  (b) Every liver has some cell as part at all times it exists.
+ - Liver instance_of material anatomical entity
+ - "Liver has_part_at_all_times Cell" (from project 3 Google Doc) 
+  
+  (c) John was a child, then an adult, then a senior. 
+- John instance_of object
+- John_childhood instance_of process
+- John_adulthood instance_of process
+- John_goldenyears instance_of process
+- JohnStory instance_of history
+...
+- JohnStory occurs_in John
+- John_childhood part_of JohnStory
+- John_adulthood part_of JohnStory
+- John_goldenyears part_of JohnStory
+...
+- John_childhood causally_upstream_of John_adulthood
+- John_adulthood causally_upstream_of John_goldenyears
+...
+- Alternatively:
+- John_childhood precedes John_adulthood
+- John_adulthood precedes John_goldenyears
+  
+  (d) Goofus and Gallant are married at each point in a three year span. 
+ - Goofus instance_of object
+ - Gallant instance_of object
+ ...
+ - 3_years instance_of one-dimensional temporal region
+ ...
+ - GooGall_marriage instance_of process
+ - Love4Goofus part_of GooGall_marriage
+ - Love4Gallant part_of GooGall_marriage
+ - Love4Goofus similar_in_magnitude_relative_to Love4Gallant
+ - GooGall_marriage occurs_in 3_years
+ ...
+ - After referencing the project 3 Google doc, I can now see it is the case that the statement does not specify Goofus and Gallant are married to each other.
+ - My work here should hopefully reflect this:
+ ...
+ - Goo_marriage instance_of process
+ - Gall_marriage instance_of process
+ - Goo_marriage occurs_in 3_years
+ - Gall_marriage occurs_in 3_years
+ - Goofus partner_in Goo_marriage
+ - Gallant partner_in Gall_marriage
+  
+```
+
+4. Using the language of First-Order Logic, represent the following natural language expressions; you are welcome to introduce new terms where needed: 
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  ∃Tues.B(S,A) ∧ Wed.B(S,¬A)
+  - 'There exists a Tuesday with Sally bodying an arm and there is a Wednesday with Sally bodying a not-arm.'
+  
+  (b) Every liver has some cell as part at all times it exists.
+  ∀Liver.C
+  - 'All Livers have a Cell'
+  - I don't know how to represent "at all times it exists" in FOL; I was thinking "all livers do not have a not-cell" but that either doesn't make sense, is redundant, or both.
+  
```

Comment:
You don't really need to explicitly model time, as long as you say that all livers have a cell. If you want to use the full expressive power of first order logic you can go with the formula on the google doc. It's not really clear here whether Liver is a property or a variable.